components/AlumniCarousel.jsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @next/next/no-img-element */
+import { ALUMNI } from "../data/alumni";
+import { Navigation } from "swiper";
+import { SwiperSlide } from "swiper/react";
+import { faLinkedin } from "@fortawesome/free-brands-svg-icons";
+import Carousel from "./Carousel";
+import FontAwesomeLink from "./FontAwesomeLink";
+import "swiper/css";
+import "swiper/css/navigation";
+
+const CAROUSEL_CONTAINER = "text-white rounded-lg w-[20rem] h-[40rem] shadow-2xl md:w-4/5 swiper-button-white";
+const ALUMNI_TILE =
+  "bg-[#7055FD] w-[20rem] h-[40rem] rounded-lg items-center p-14 lg:p-10 flex flex-col shadow-2xl md:w-full md:flex-row";
+const ALUMNI_PROFILE = "flex flex-col items-center mx-auto";
+
+const SPACE_BETWEEN_ELEMENTS = 50;
+
+const AlumniTile = ({ name, file, position, prevRole, testimonial, linkedin }) => {
+  return (
+    <div className={ALUMNI_TILE}>
+      <div className={ALUMNI_PROFILE}>
+        <img src={`/profiles/${file}`} alt="Profile Picture" className="w-50 mb-3 mt-5 rounded-full md:w-80 mx-auto" />
+        <h2 className="text-[2rem] md:text-[2.5rem] font-bold">{name}</h2>
```

Comment:
There dont seem to centered

<img width="254" alt="Screen Shot 2022-08-23 at 10 14 59 PM" src="https://user-images.githubusercontent.com/47438886/186334615-ca0047f6-7d6b-4153-b9cf-a0cc35a685f2.png">

Reply:
Check now